public/css/bootstrap/bootstrap.min.css

```diff
@@ -0,0 +1,5 @@
+/*!
```

Comment:
we dont need to import bootsrap again as its added to laravel by default. see app.css